cli/pipeline/template.go

```diff
@@ -104,6 +105,7 @@ spec:
     -
       name: paddle
       image: "219541440308.dkr.ecr.eu-west-1.amazonaws.com/paddlecontainer:latest"
+      imagePullPolicy: Always
```

Comment:
This is not needed, ':latest' will always pull: https://kubernetes.io/docs/concepts/configuration/overview/#container-images

Reply:
Yes, I know. Hence the name of the commit: "specify imagepullpolicy for explicitness" :)